src/js/_enqueues/admin/common.js

```diff
@@ -931,7 +931,7 @@ $( function() {
 			adjustment = maxtop;
 		}
 
-		if ( adjustment > 1 ) {
+		if ( adjustment > 1 && $window.width() > 783 ) {
```

Comment:
Could you edit this to `$window.width() > 782`?

The menu should behave the same at 783 pixels as at 784 and wider.

Reply:
Sorry, my mistake

Reply:
I correct the width